Add instructions for potential neovide users (#183)
LunarVim/lunarvim.org: docs/01-installing.md

```
Reference: [WSL automount settings](https://docs.microsoft.com/en-us/windows/wsl/wsl-config#automount-settings)

## Tips for Neovide users
In order to use lunarvim with [Neovide](https://github.com/neovide/neovide) you need to add the following line to your `config.lua`

```lua
vim.o.guifont = "Your font name"
```

> Where your font name is the literal name of the font you want to use. For example `Inconsolata Nerd Font`.

Author: ```suggestion
Where `Your Font Name` is the literal name of the font you want to use. For example `"Inconsolata Nerd Font"`.
```

> You will also need to export the right environment variables and call neovide with the apropriate parameters. An easy way to achieve this is to create a little script like this, for example `~/.local/bin/lvim-gui`

Author: ```suggestion
You will also need to export the right environment variables and call neovide with the apropriate parameters. An easy way to achieve this is to create a little executable script like, for example `~/.local/bin/lvim-gui`
```

> You will also need to export the right environment variables and call neovide with the apropriate parameters. An easy way to achieve this is to create a little script like this, for example in your home directory (`~/neovide`)

Reviewer: ```suggestion
You will also need to export the right environment variables and call neovide with the apropriate parameters. An easy way to achieve this is to create a little script like this, for example `~/.local/bin/lvim-gui`
```

```bash
#!/bin/sh

> # ~/neovide

Reviewer: ```suggestion
# ~/.local/bin/lvim-gui
```

> export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-"/Users/danielo/.local/share/lunarvim"}"
> export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-"/Users/danielo/.config/lvim"}"
> export LUNARVIM_CACHE_DIR="${LUNARVIM_CACHE_DIR:-"/Users/danielo/.cache/nvim"}"

Reviewer: ```suggestion
export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-"$HOME/.local/share/lunarvim"}"
export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-"$HOME/.config/lvim"}"
export LUNARVIM_CACHE_DIR="${LUNARVIM_CACHE_DIR:-"$HOME/.cache/lvim"}"
```

exec neovide -- -u "$LUNARVIM_RUNTIME_DIR/lvim/init.lua" "$@"
```

Then give it the proper execution permissions 

```bash

> chmod +x ~/neovide

Reviewer: ```suggestion
chmod +x ~/.local/bin/lvim-gui
```

```

And you can launch from now on like this:

```bash

> ~/neovide

Reviewer: ```suggestion
lvim-gui
```

# or with params

> ~/neovide some-file

Reviewer: ```suggestion
lvim-gui some-file
```
